README.md

```diff
@@ -96,6 +96,9 @@ You can use a custom authorizer class by specifying a configure block in an init
 ```ruby
 JSONAPI::Authorization.configure do |config|
   config.authorizer = MyCustomAuthorizer
+  config.pundit_user = :user
+  # or
+  config.pundit_user = ->(context){ context[:current_user] }
```

Comment:
These changes could be described a little better, as why they are necessary and that they aren't required even if you're using a custom authorizer. I can do that once this PR is merged as well :+1: